files/plugins/process_check_host.py

```diff
@@ -59,7 +61,13 @@ def get_processes(parent_pid=None):
         procs = [x for x in psutil.process_iter()
                  if x.pid not in excluded_pids]
     else:
-        procs = [x for x in psutil.Process(parent_pid).get_children()]
+        # psutil 1.2.1 has a children() method for the process object
+        # but the latest psutil uses get_children(). We need to handle
+        # this carefully.
```

Comment:
Ya, that's the general way of handling it.  Could also look for the method using an if statement instead of falling back to try/except (which could catch other bad behavior where we don't want it to).